src/components/pt-element/PtElementInfo.test.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from "react";
+import PtElementInfo from "./PtElementInfo";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+const IRON_ELEMENT = {
+  atomic: 26,
+  symbol: "Fe",
+  name: "Iron",
+  realname: "Iron",
+  atomicMass: "55.845",
+  cpkHexColor: "E06633",
+  electronicConfiguration: "[Ar] 3d6 4s2",
+  electronegativity: 1.83,
+  atomicRadius: 156,
+  ionRadius: "78 (+2*)",
+  vanDelWaalsRadius: null,
+  ionizationEnergy: 763,
+  electronAffinity: -16,
+  oxidationStates: "-2, -1, 1, 2, 3, 4, 5, 6",
+  standardState: "solid",
+  bondingType: "metallic",
+  meltingPoint: "1808",
+  boilingPoint: "3023",
+  density: 7.87,
+  group: "transition_metals",
+  yearDiscovered: null,
+  valency: "+2 +3",
+  wrongValences: ["+1 +3", "+1 +2", "+2 +4", "+1 +3 +5", "+2 +4 +6", "-1"],
+  testState: {
+    valencesTest: true,
+    ptTest: true,
+  },
+};
+
+describe("PTElementInfo", () => {
+  it("shows the elements atomic number", () => {
+    render(<PtElementInfo element={IRON_ELEMENT} />);
+
+    expect(screen.getByText(IRON_ELEMENT.atomic)).toBeInTheDocument();
+  });
+
+  it("shows the elements symbol", () => {
+    render(<PtElementInfo element={IRON_ELEMENT} />);
+
+    expect(screen.getByText(IRON_ELEMENT.symbol)).toBeInTheDocument();
+  });
+
+  it("shows the elements name", () => {
+    render(<PtElementInfo element={IRON_ELEMENT} />);
+
+    expect(screen.getByText(IRON_ELEMENT.name)).toBeInTheDocument();
+  });
+
+  it("adds the element group as a classname", () => {
```

Comment:
This test feels a bit odd, but I don't have any better suggestions for you.

Reply:
yeah, it feels weird, but I can't think of a better alternative besides not testing it